pkg/test/echo/docker/Dockerfile.app_sidecar

```diff
@@ -9,6 +9,7 @@ COPY certs/default/*                  /var/run/secrets/istio/
 # Install the sidecar components
 COPY istio-sidecar.deb  /tmp/istio-sidecar.deb
 RUN dpkg -i /tmp/istio-sidecar.deb && rm /tmp/istio-sidecar.deb
+RUN sed -i "s/PROV_CERT=/#PROV_CERT=/g" /var/lib/istio/envoy/sidecar.env
```

Comment:
what is this for? This seems odd - it could at least use a comment if we do need it

Reply:
Because the setting in sidecar.env will override env var we set in https://github.com/istio/istio/blob/master/pkg/test/framework/components/echo/docker/workload.go.
So we need to make sure we do not specify PROV_CERT for local testing